tests/Operation/FindFunctionalTest.php

```diff
@@ -123,6 +125,45 @@ public function provideTypeMapOptionsAndExpectedDocuments()
         ];
     }
 
+    public function testMaxAwaitTimeMS()
+    {
+        $maxAwaitTimeMS = 10;
+
+        // Create a capped collection.
+        $databaseName = $this->getDatabaseName();
+        $cappedCollectionName = $this->getCollectionName() . '.capped';
+        $cappedCollectionOptions = [
+            'capped' => true,
+            'max' => 100,
+            'size' => 1048576,
+        ];
+
+        $operation = new CreateCollection($databaseName, $cappedCollectionName, $cappedCollectionOptions);
+        $operation->execute($this->getPrimaryServer());
+
+        // Insert documents into the capped collection.
+        $cappedNamespace = $databaseName . '.' . $cappedCollectionName;
+        $bulkWrite = new BulkWrite(['ordered' => true]);
+        $bulkWrite->insert(['_id' => 1]);
+        $result = $this->manager->executeBulkWrite($cappedNamespace, $bulkWrite);
+
+        // Get a cursor of type TAILABLE_AWAIT.
+        $operation = new Find($databaseName, $cappedCollectionName, [], ['cursorType' => 3, 'maxAwaitTimeMS' => $maxAwaitTimeMS]);
+        $cursor = $operation->execute($this->getPrimaryServer());
+        $it = new \IteratorIterator($cursor);
+
+        // Make sure we await results for no more than the maxAwaitTimeMS.
+        $it->rewind();
+        $it->next();
+        $startTime = microtime(true);
+        $it->next();
+        $this->assertLessThan($maxAwaitTimeMS, (microtime(true) - $startTime));
```

Comment:
It's likely that this assertion will always succeed. In the float returned by [`microtime(true)`](http://php.net/manual/en/function.microtime.php), the digits to the right of the decimal point indicate fractions of a second with microsecond precision.

If you're subtracting `$startTime` from the end time here, we should check that the difference is _at least_ ten milliseconds (i.e. `0.01`) to assert that `IteratorIterator::next()` blocked for `maxAwaitTimeMS` plus any additional network latency.

We can either change this to `$this->assertGreaterThanOrEqual($maxAwaitTimeMS * 0.001, ...)` or you can you the literals `10` and `0.01` in the Find options and this assertion, respectively.